prov/verbs/src/verbs_ep.c

```diff
@@ -876,6 +912,14 @@ int fi_ibv_open_ep(struct fid_domain *domain, struct fi_info *info,
 	ep->util_ep.ep_fid.fid.ops = &fi_ibv_ep_ops;
 	ep->util_ep.ep_fid.ops = &fi_ibv_ep_base_ops;
 
+	if (!ep->inject_limit) {
+		ep->util_ep.ep_fid.msg->inject = fi_no_msg_inject;
+		ep->util_ep.ep_fid.msg->injectdata = fi_no_msg_injectdata;
+		ep->util_ep.ep_fid.rma->inject = fi_no_rma_inject;
+		ep->util_ep.ep_fid.rma->injectdata = fi_no_rma_injectdata;
+	}
```

Comment:
Hmm. Wouldn't this disallow using fi_inject for sending 0 byte messages but with CQ data?

Reply:
We can't use `ibv_post_send` with `INLINE` operational flag, if verbs driver reports supported inline data size = 0 (or Verbs provider requested it thru the capabilities -- can be done by settign `FI_VERBS_INLINE_SIZE`).